From: Rosen Penev <rosenp@gmail.com>
Date: Fri, 5 Mar 2021 00:10:05 -0800
Subject: meson: update to 0.56.2

This is the last version supporting Python 3.5, which is still in wide
use. Going forward, python3/host would need to be used. Not sure if
worth it.

Remove paths section as it's deprecated.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

diff --git a/devel/meson/Makefile b/devel/meson/Makefile
index 68b8093c3b3c592e763a929eea52b60e9cafebce..b90f511afe846a438ceef8551d6c80bb605d4517 100644
--- a/devel/meson/Makefile
+++ b/devel/meson/Makefile
@@ -1,11 +1,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=meson
-PKG_VERSION:=0.56.0
+PKG_VERSION:=0.56.2
 PKG_RELEASE:=1
 
 PYPI_NAME:=meson
-PKG_HASH:=291dd38ff1cd55fcfca8fc985181dd39be0d3e5826e5f0013bf867be40117213
+PKG_HASH:=3cb8bdb91383f7f8da642f916e4c44066a29262caa499341e2880f010edb87f4
 
 PKG_MAINTAINER:=Andre Heider <a.heider@gmail.com>
 PKG_LICENSE:=Apache-2.0
diff --git a/devel/meson/src/openwrt-cross.txt.in b/devel/meson/src/openwrt-cross.txt.in
index 32e4ca48dabba7232cf435652e7e08909aef89a4..30b4c116c413fdd5cd5159d643c78a720e0cacc2 100644
--- a/devel/meson/src/openwrt-cross.txt.in
+++ b/devel/meson/src/openwrt-cross.txt.in
@@ -11,6 +11,7 @@ c_args = [@CFLAGS@]
 c_link_args = [@LDFLAGS@]
 cpp_args = [@CXXFLAGS@]
 cpp_link_args = [@LDFLAGS@]
+prefix = '/usr'
 
 [host_machine]
 system = 'linux'
@@ -20,6 +21,3 @@ endian = '@ENDIAN@'
 
 [properties]
 needs_exe_wrapper = true
-
-[paths]
-prefix = '/usr'
diff --git a/devel/meson/src/openwrt-native.txt.in b/devel/meson/src/openwrt-native.txt.in
index 601edfbc4666dcf35783e0cca0e0b70ff19c6e2d..50308ec8e433b1d9ba4c25ef492057edde00f0d4 100644
--- a/devel/meson/src/openwrt-native.txt.in
+++ b/devel/meson/src/openwrt-native.txt.in
@@ -8,8 +8,6 @@ c_args = [@CFLAGS@]
 c_link_args = [@LDFLAGS@]
 cpp_args = [@CXXFLAGS@]
 cpp_link_args = [@LDFLAGS@]
-
-[paths]
 prefix = '@PREFIX@'
 sbindir = 'bin'
 libdir = 'lib'
